lib/liquid_voting/voting.ex

```diff
@@ -24,31 +25,44 @@ defmodule LiquidVoting.Voting do
 
   """
   def create_vote(attrs \\ %{}) do
-    Repo.transaction(fn ->
-      # TODO: refactor case statements into small functions.
+    Tracer.with_span "#{__MODULE__} #{inspect(__ENV__.function)}" do
+      Tracer.set_attributes([
+        {:request_id, Logger.metadata()[:request_id]},
+        {:vars,
```

Comment:
`vars` is too vague here, I think. How about `params`? That's more informative about the vars being the ones submitted in the request.